src/Model/User.php

```diff
@@ -313,7 +313,7 @@ public static function getByNickname($nickname, array $fields = [])
 	public static function getIdForURL(string $url)
 	{
 		// Avoid any database requests when the hostname isn't even part of the url.
```

Comment:
Please update this comment to mention the local node hostname.
```suggestion
		// Avoid database queries when the local node hostname isn't even part of the url.
```